### DISCUSSION TITLE
Does Liberation assign known SAM sites and their threat ranges to SA pages?

### DISCUSSION THREAD

**Discussion by jubuttib** [Khopa/dcs_liberation#1606] [Q&A] [answered]
I'm not entirely sure how to formulate this question, since I'm not yet well versed with how the whole thing actually works with the F/A-18C, F-16C and ME, but here's an attempt:

Based on Chuck's guides for the Hornet and Viper, both these planes have the capability to show known SAM sites and their estimated threat ranges on the Situational Awareness page of the displays, and that the feature is loaded using the DTC via the Mission Editor, rather than being dynamic in-game.

Does Liberation currently support, and do this automatically?

**Comment by Mustang-25** [comment, marked as answer]
The DTC feature does not exist for the DCS Hornet or Viper, SAM threat rings are automatically displayed by default. If you load into any campaign with SAMs you will see the threat rings.

**Reply by jubuttib** [reply]
Ah interesting, thanks for the reply!

Is there any kind of logic to what SAM sites are shown, or are all of them just automatically there? Like they need to be spotted by friendly units, or something? Kinda seems like it would make finding them rather trivial if they all just automatically pop up.

**Reply by Mustang-25** [reply]
They are all shown. The way it works in the ME, it's just a checkbox for shown or hidden. There is an open feature request for an intelligence system that will address what gets shown and what does not.

**Reply by jubuttib** [reply]
Thanks for confirming this. I'll stay away from the two planes then, because honestly perfect ELINT isn't something I want in my game.
